Phonebook.Backend/Phonebook.Backend.PictureService/Phonebook.Backend.PictureService/Helpers/HelpersThing.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Phonebook.Backend.PictureService.Helpers
+{
+    public static class HelpersThing

```

Comment:
Can we rename this? You refactored all the way. I think this class deserves a better name :)

Reply:
thats done with [ec1d6c1](https://github.com/T-Systems-MMS/phonebook/pull/161/commits/ec1d6c1641628e62a86a96d1034f1bc0375f045b)